tests/conftest.py

```diff
@@ -57,3 +57,31 @@ def tmp_another_package(root, tmp_empty):
     yield tmp_empty
 
     os.chdir(old)
+
+
+@pytest.fixture
+def tmp_package_modi(root, tmp_empty):
+    old = os.getcwd()
+    path_to_templates = root / "tests" / "assets" / "package_name"
+    shutil.copytree(str(path_to_templates), "copy")
+    os.chdir("copy")
+
+    subprocess.run(["git", "init"])
+    subprocess.check_call(["git", "config", "commit.gpgsign", "false"])
+    subprocess.check_call(["git", "config", "user.email", "ci@ploomberio"])
+    subprocess.check_call(["git", "config", "user.name", "Ploomber"])
+    subprocess.run(["git", "checkout", "-b", "main"])
+    subprocess.run(["git", "add", "--all"])
+    subprocess.run(["git", "commit", "-m", "init-commit-message"])
+
+    subprocess.run(["git", "checkout", "-b", "test_modified_doc"])
+    subprocess.run(["mkdir", "-p", "test_doc1"])
```

Comment:
I have converted all the calls to `pathlib.Path`, but Windows tests are still failing. For more info, check the tests [here](https://github.com/ploomber/pkgmt/actions/runs/5147192131/jobs/9267273488?pr=49#step:6:28)

Reply:
yeah, some stuff isn't compatible with windows, so no need to add windows and macos to the testing configuration. I suggested changing it to pathlib because at some point we might want to support those OS as well

Reply:
So, I commented all those tests, but `test_check_modified` is failing. It seems to me more of a `git` issue because the `debug` output says `modified.txt` has been deleted and a new file `modified.txt` has been created. (Output [here](https://github.com/ploomber/pkgmt/actions/runs/5147192131/jobs/9267273488?pr=49#step:6:77))

So, is it ok for now to skip this part, the part of debugging windows test, I have already added converted the calls to `path.Pathlib`?

Reply:
I think it maybe because of not using `' '` in exclude path like `':^doc'` instead of `:^doc` (due to this [error](https://github.com/ploomber/pkgmt/actions/runs/5157612645/jobs/9290253296#step:6:231)), I will try this approach tomorrow

Reply:
Nope not working
[Github Actions](https://github.com/ploomber/pkgmt/actions/runs/5157612645)

Some links: 
[Link1](https://github.com/ploomber/pkgmt/actions/runs/5157612645/jobs/9290253296#step:6:214)
[Link2](https://github.com/ploomber/pkgmt/actions/runs/5157612645/jobs/9290253296#step:6:231)